Modules/Core/Common/include/itkVector.h

```diff
@@ -106,7 +106,7 @@ class ITK_TEMPLATE_EXPORT Vector : public FixedArray<T, VVectorDimension>
    * \note The other five "special member functions" are defaulted implicitly, following the C++ "Rule of Zero". */
   Vector() = default;
 
-#if !defined(ITK_LEGACY_FUTURE_REMOVE)
+#if !defined(ITK_LEGACY_REMOVE)
```

Comment:
Do we need an entry in migration guide for this change? Or is this minor enough?